packages/fxa-auth-server/lib/senders/emails/partials/appBadges/en.ftl

```diff
@@ -4,5 +4,18 @@ body-android-badge = <img data-l10n-name="google-play-badge" alt="Download { $pr
 # Variables:
 #  $productName (String) - The name of the product to be downloaded, e.g. Mozilla VPN, or Firefox
 body-ios-badge = <img data-l10n-name="apple-app-badge" alt="Download { $productName } on the { -app-store }">
-another-desktop-device = Or, install on <a data-l10n-name="anotherDeviceLink">another desktop device</a>.
-another-device = Or, install on <a data-l10n-name="anotherDeviceLink">another device</a>.
+# Variables:
+#  $productName (String) - The name of the product to be downloaded, e.g. Mozilla VPN, or Firefox
+another-desktop-device-2 = Or, install { $productName } on <a data-l10n-name="anotherDeviceLink">another desktop device</a>.
+# Variables:
+#  $productName (String) - The name of the product to be downloaded, e.g. Mozilla VPN, or Firefox
+another-device-2 = Or, install { $productName } on <a data-l10n-name="anotherDeviceLink">another device</a>.
```

Comment:
Per the [doc section 1.2](https://docs.google.com/document/d/18gF_7Ux-jE_qfEFciOcPml0-G1SfHkjiJSMPwh2hnSc/edit#heading=h.q439jqhxcydz), I think this is now suppsoed to say `Install`, not `Or, install`.